auto_update_tests.py

```diff
@@ -235,7 +235,7 @@
     f.close()
     cmd = ['mozjs', 'a.js']
     out = run_command(cmd, stderr=subprocess.STDOUT)
-    open(os.path.join('test', 'binaryen.js', s + '.txt'), 'w').write(out)
+    with open(os.path.join('test', 'binaryen.js', s + '.txt'), 'w') as o: o.write(out)
```

Comment:
Every other instance of `with` in this file uses this pattern. I think it may have been a compromise between `you should use with instead of just open().write!' and 'but verbosity!'